src/Theme.cpp

```diff
@@ -198,6 +198,11 @@ static const unsigned char HiContrastImageCacheAsData[] = {
 #include "HiContrastThemeAsCeeCode.h"
 };
 
+
+static const unsigned char protools_template_iconImageCacheAsData[] = {
+#include "ProToolsThemeAsCeeCode.h"
```

Comment:
You have to rename the variable to ProToolsImageCacheAsData because that's what you specified in code.

Reply:
Should work fine now. Sorry about that.